custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -143,12 +143,21 @@ def _init_mlops(self):
         self._mlops.init()
 
     def _configure_mlops_for_chat(self):
-        self._mlops.set_channel_config("spooler_type=API")
+        # If monitor_settings were provided (e.g. for testing) use them, otherwise we will
+        # use the API spooler as the default config.
+        if self._params.get("monitor_settings"):
+            self._mlops.set_channel_config(self._params["monitor_settings"])
+        else:
+            self._mlops.set_api_spooler(
+                # TODO: when 10.2.0 has been released...
+                # mlops_service_url=self._params["external_webserver_url"],
+                # mlops_api_token=self._params["api_token"],
+            )
 
-        self._prompt_column_name = self._get_prompt_column_name()
+        self._prompt_column_name = self.get_prompt_column_name()
         logger.debug("Prompt column name: %s", self._prompt_column_name)
 
-    def _get_prompt_column_name(self):
+    def get_prompt_column_name(self):
```

Comment:
Why make it public? It's only called here?

Reply:
I want to call it in the lower level predictors. Right now we use a runtime param to specify the prompt column -- I didn't know about this method. I think I will use this method instead.